src/components/sortableList/SortableList.api.json

```diff
@@ -43,11 +43,55 @@
     }
   ],
   "snippet": [
-    "<SortableList",
-    "  data={data$1}",
-    "  onOrderChange={onOrderChange$2}",
-    "  renderItem={renderItem$3}",
-    "  keyExtractor={keyExtractor$4}",
-    "/>"
+    "function Example(props) {",
+    "  const data = Array.from({length: 30}, (_, index) => {",
```

Comment:
Let's change the length to something smaller so it won't take up the whole screen space